src/core/stdc/config.d

```diff
@@ -104,31 +104,28 @@ version( Windows )
     enum __c_long  : int;
     enum __c_ulong : uint;
 
-    alias __c_long   cpp_long;
-    alias __c_ulong  cpp_ulong;
-
     alias int   c_long;
     alias uint  c_ulong;
 
+    alias __c_long   cpp_long;
+    alias __c_ulong  cpp_ulong;
+
```

Comment:
Why is this just moved a few lines?

Reply:
In order to comply with the DDoc order, which is the intuitive one: C long, C++ long, C++ longlong.